deployer/pipe-listener.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+# shellcheck disable=SC2046
+while true; do (cd /var/docker/setup; ./interactive_setup.sh $(cat deployer/arg-pipe) &> deployer/deploy-log.txt); done
```

Comment:
Can we get our security expert to try and break this via the deployer-backend? I haven't invested time into possible hacks here yet.